### PR TITLE
Amls 5479 - Removal of purchase content

### PR DESCRIPTION
**Pull request by matthewharris1**
Content changes for AMP service - File names have also changed to remove reference to "bought"

## How Has This Been Tested?
Unit
Acceptance tests - Required
Performance tests - Required

## Checklist

- [ ] Breaking change (fix or feature that would cause existing functionality to change).
- [x] Build passing.
- [x] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [x] Requires acceptance test run.
- [x] Appropriate labels added.
- [ ] RELEASE NOTES ADDED.

### CODE REVIEW

File: app/utils/CheckYourAnswersHelper.scala [hunk 1]
```diff
@@ -82,12 +82,12 @@ class CheckYourAnswersHelper(userAnswers: UserAnswers)(implicit messages: Messag
       )
   }
 
-  def boughtOrSoldOverThreshold: Option[AnswerRow] = userAnswers.get(BoughtOrSoldOverThresholdPage) map {
+  def boughtOrSoldOverThreshold: Option[AnswerRow] = userAnswers.get(SoldOverThresholdPage) map {
```
Comment (slavvco) [inline app/utils/CheckYourAnswersHelper.scala]: boughtOrSoldOverThreshold ?

File: conf/messages.en [hunk 2]
```diff
@@ -97,27 +97,27 @@ typeOfParticipantDetail.error.required = Enter the other type of art market part
...
-dateTransactionOverThreshold.error.required = Date of the first sale or purchase must include a {0}
-dateTransactionOverThreshold.error.invalid = Enter a real date for the first sale or purchase
-dateTransactionOverThreshold.error.startdate = Date of the first sale or purchase must be after 10 January 2020
-dateTransactionOverThreshold.error.future = Date of the first sale or purchase must be in the past
+SoldOverThreshold.title = Has your business made a sale of art for €10,000 or more after 10 January 2020?
+SoldOverThreshold.heading = Has your business made a sale of art for €10,000 or more after 10 January 2020?
+SoldOverThreshold.checkYourAnswersLabel = Has your business made a sale of art for €10,000 or more after 10 January 2020?
+SoldOverThreshold.error.required = Select yes if your business made a sale of art for €10,000 or more after 10 January 2020
+SoldOverThreshold.hint = If you do not use euros, you’ll need to convert the amount to the currency you use.
+
```
Comment (slavvco) [inline conf/messages.en]: SoldOverThreshold -> no need for capital S

File: app/utils/CheckYourAnswersHelper.scala (hunk 1 above)
Comment (matthewharris1) [reply app/utils/CheckYourAnswersHelper.scala]: Done

File: conf/messages.en (hunk 2 above)
Comment (matthewharris1) [reply conf/messages.en]: Done

**Pull request merged**